x-pack/plugins/enterprise_search/public/applications/__mocks__/kea.mock.ts

```diff
@@ -145,4 +145,31 @@ export class LogicMounter {
       ? (listeners as Function)(listenersArgs) // e.g., listeners({ values, actions, props }) => ({ ... })
       : listeners; // handles simpler logic files that just define listeners: { ... }
   };
+
+  /**
+   * Small QOL helper for calling a listener directly derived from getListeners()
+   * and automatically passing mocking breakpoint() as well as invoking the
+   * listener action for test coverage.
+   *
+   * Example usage:
+   *
+   * const { mount, getListeners, invokeAsyncListener } = new LogicMounter(SomeLogic);
+   *
+   * it('some test', async () => {
+   *   mount();
+   *   const { someListener } = getListeners({ values: { someMockValue: false } });
+   *
+   *   await invokeAsyncListener(someListener, { someMockArgument: true });
+   * })
+   */
+  public invokeAsyncListener = async (listenerFn: Function, listenerArgs: unknown) => {
```

Comment:
also not 100% in love with the name of this, maybe `invokeBreakpointListener` is better or more specific?